app/src/org/commcare/network/HttpCalloutTask.java

```diff
@@ -38,7 +38,7 @@ public enum HttpCalloutOutcomes {
         BadResponse,
         AuthFailed,
         UnknownError,
-        BadCertificate,
+        BadSSLCertificate,
```

Comment:
nit: call it `BadSslCertificate`

Reply:
Sounds good. There are three enums that had the BadSSLCertificate value (HttpCalloutTask, StockMessages, and AppInstallStatus), I changed all three to use BadSslCertificate instead.